src/components/organisms/UiHorizontalPaging/stories/MobileMenu.vue

```diff
@@ -0,0 +1,163 @@
+        </UiHeading>
+      </div>
+    </template>
+    <UiHorizontalPaging
+      ref="horizontalPagingTemplateRefs"
+      v-model="value"
+      v-bind="args"
+      :menu-template-refs="menuTemplateRefs"
+    >
+      <template
+        #menu="{
+          items,
+          isActive,
+        }"
+      >
+        <div
+          :class="[
+            'mobile-menu__menu',
+            { 'mobile-menu__menu--is-hidden': isActive },
+          ]"
+        >
+          <UiMenu
+            ref="menuTemplateRefs"
+            :items="items"
+          />
+          <footer class="mobile-menu__footer">
+            <UiLink
+              href="https://infermedica.com/"
+              target="_blank"
+              class="ui-link--theme-secondary"
+            >
+              © 2021 Infermedica
+            </UiLink>
+          </footer>
+        </div>
+      </template>
+      <template #medical-certification>
+        <MedicalCertification />
+      </template>
+      <template #instruction-for-use>
+        <UiLoader type="skeleton" />
+      </template>
+      <template #terms-of-service>
+        <TermsOfService />
+      </template>
+      <template #privacy-policy>
+        <UiLoader type="skeleton" />
+      </template>
+      <template #interview-id>
+        <InterviewId />
+      </template>
+    </UiHorizontalPaging>
+  </UiSidePanel>
+</template>
+
+<script setup>
+import {
+  ref,
+  computed,
+  useAttrs,
+  defineOptions,
+  inject,
+  nextTick,
+} from 'vue';
+import {
+  UiHorizontalPaging,
+  UiMenu,
+  UiLoader,
+  UiSidePanel,
+  UiButton,
+  UiIcon,
+  UiLink,
+  UiHeading,
+} from '@infermedica/component-library';
+import MedicalCertification from './_internal/MedicalCertification.vue';
+import TermsOfService from './_internal/TermsOfService.vue';
+import InterviewId from './_internal/InterviewId.vue';
+import { focusElement } from '../../../../utilities/helpers';
+
+defineOptions({ inheritAttrs: false });
+const attrs = useAttrs();
+const args = computed(() => (attrs));
+const value = inject('value', []);
+
+const horizontalPagingTemplateRefs = ref(null);
+const isActive = computed(() => (horizontalPagingTemplateRefs.value?.isActive));
+const sidePanelTitle = computed(() => (horizontalPagingTemplateRefs.value?.currentTitle));
+const backToTitle = computed(() => (horizontalPagingTemplateRefs.value?.backToTitle));
+const menuTemplateRefs = ref(null);
+const handleBackClick = async () => {
+  horizontalPagingTemplateRefs.value.handleBackClick();
+  await nextTick();
+  focusElement(menuTemplateRefs.value.lastFocusedMenuItemTemplateRefs.itemTemplateRefs.content.$el);
+};
+</script>
+
+<style lang="scss">
+@use "../../../../styles/mixins";
+
+.mobile-menu {
+  $element: mobile-menu;
+
+  --side-panel-content-padding-block: 0;
+  --side-panel-content-padding-inline: 0;
+
+  &__title {
+    display: flex;
+    gap: var(--space-12);
+  }
+
+  &__back {
+    align-self: flex-start;
+    margin-block-start: 3px;
```

Comment:
not variable?

Reply:
Nope, it's a design hack. Chevron is justifying to the top but it needs `3px` spacing.
![Zrzut ekranu 2024-05-7 o 14 53 48](https://github.com/infermedica/component-library/assets/12138170/c753a5a4-52f2-4185-8e64-175d51098a75)